content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,675 @@
+```
+
+Use ```kubectl``` to forward the port:
+```shell
+kubectl port-forward service/configmap-service 8080:8080 & # this stays running in the background
+```
+
+Access the service.
+```shell
+curl http://localhost:8080
+```
+
+You should see an output similar to:
+```
+Fri Jan  5 08:08:22 UTC 2024 My preferred color is red
+```
+
+Edit the ConfigMap:
+```shell
+kubectl edit configmap color
+```
+
+In the editor that appears, change the value of key `color` from `red` to `blue`. Save your changes.
+The kubectl tool updates the ConfigMap accordingly (if you see an error, try again).
+
+Here's an example of how that manifest could look after you edit it:
+
+```yaml
+apiVersion: v1
+data:
+  color: blue
+kind: ConfigMap
+# You can leave the existing metadata as they are.
+# The values you'll see won't exactly match these.
+metadata:
+  creationTimestamp: "2024-01-05T08:12:05Z"
+  name: color
+  namespace: configmap
+  resourceVersion: "1801272"
+  uid: 80d33e4a-cbb4-4bc9-ba8c-544c68e425d6
+```
+
+Loop over the service URL for few seconds. 
+```shell
+# Cancel this when you're happy with it (Ctrl-C)
+while true; do curl --connect-timeout 7.5 http://localhost:8080; sleep 10; done
+```
+
+You should see the output change as follows:
+```
+Fri Jan  5 08:14:00 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:02 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:20 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:22 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:32 UTC 2024 My preferred color is blue
+Fri Jan  5 08:14:43 UTC 2024 My preferred color is blue
+Fri Jan  5 08:15:00 UTC 2024 My preferred color is blue
+```
+
+## Update configuration via a ConfigMap in a Pod possessing a sidecar container {#rollout-configmap-sidecar}
+
+The above scenario can be replicated by using a [Sidecar Container](/docs/concepts/workloads/pods/sidecar-containers/) as a helper container to write the HTML file.  
+As a Sidecar Container is conceptually an Init Container, it is guaranteed to start before the main webserver container.  
+This ensures that the HTML file is always available when the webserver is ready to serve it.  
+Please see [Enabling sidecar containers](/docs/concepts/workloads/pods/sidecar-containers/#enabling-sidecar-containers) to utilize this feature.
+
+If you are continuing from the previous scenario, you can reuse the ConfigMap named `color` for this scenario.  
+If you are executing this scenario independently, use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap color --from-literal=color=blue
+```
+
+Below is an example manifest for a Deployment that manages a set of Pods, each with a main container and a Sidecar Container. The two containers share an `emptyDir` volume that they use to communicate.
```

Comment:
Sidecar should be spelt as "sidecar" ?

Reply:
Ack. This will be handled in the next commit.